constants/chatbotCommands.py

```diff
@@ -129,6 +129,38 @@ async def _help(fro: str, chan: str, message: list[str]) -> str:
     return "\n".join(l)
 
 
+@command(
+    trigger="!addbn",
+    syntax="<name>",
+    privs=privileges.ADMIN_MANAGE_PRIVILEGES,
+)
+async def addbn(fro: str, chan: str, message: list[str]) -> str:
+    """Add BN priveleges to a user"""
+    username = message[0]
```

Comment:
indexing an empty list can cause an error, so we should validate that the user _actually_ sent a message

to do this we can do:
```py
if not message:
    return "Invalid command syntax"
```

There might be a few examples around